src/utils.rs

```diff
@@ -13,7 +13,7 @@ use crate::error::Error;
 use serde::de::DeserializeOwned;
 use serde::Serialize;
 use std::fs::File;
-use std::io::{self, BufReader, BufWriter};
+use std::io::{BufReader, BufWriter};
```

Comment:
Why was this necessary

Reply:
It's not used on non-T1/mobile platforms (see the `project_dir` impl below)